apps/wallet/src/ui/app/components/alert/index.tsx

```diff
@@ -1,36 +1,47 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { Info16, Check32 } from '@mysten/icons';
-import cl from 'classnames';
+import { Info16, CheckStroke16 } from '@mysten/icons';
+import { cva, type VariantProps } from 'class-variance-authority';
 
 import LoadingIndicator from '_components/loading/LoadingIndicator';
 
 import type { ReactNode } from 'react';
 
-import st from './Alert.module.scss';
+const alertStyles = cva(
+    'rounded-2xl text-pBodySmall font-medium flex flex-row flex-nowrap justify-center items-center py-2 px-2.5 gap-2',
+    {
+        variants: {
+            mode: {
+                warning:
+                    'border-solid border bg-issue-light border-issue-dark/20 text-issue-dark',
+                success:
+                    'border-solid border bg-success-light border-success-dark/20 text-success-dark',
+                loading: 'bg-steel text-white border-warning-dark/20',
+            },
+        },
+        defaultVariants: {
+            mode: 'warning',
+        },
+    }
+);
+
+export interface AlertProps extends VariantProps<typeof alertStyles> {
+    children?: ReactNode;
+    mode?: 'warning' | 'loading' | 'success';
+}
 
-type ModeType = 'warning' | 'loading' | 'success';
-export type AlertProps = {
-    children: ReactNode | ReactNode[];
-    className?: string;
-    mode?: ModeType;
-};
 const modeToIcon = {
-    warning: <Info16 />,
-    success: <Check32 />,
+    warning: <Info16 className="h-3.5 w-3.5" />,
+    success: <CheckStroke16 className="h-3 w-3" />,
     loading: <LoadingIndicator color="inherit" />,
 };
 
-export default function Alert({
-    children,
-    className,
-    mode = 'warning',
-}: AlertProps) {
+export default function Alert({ children, mode = 'warning' }: AlertProps) {
     return (
-        <div className={cl(st.container, st[mode], className)}>
+        <div className={alertStyles({ mode })}>
             {modeToIcon[mode]}
-            <div className={st.message}>{children}</div>
+            <div className="break-all flex flex-1">{children}</div>
```

Comment:
does this need to be a `flex` container?

Reply:
I just moved over the scss style